From d14bd56baa06140f49e6f81113cf346a10b134bf Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Fri, 4 Oct 2019 18:25:17 +0000
Subject: [PATCH] MANTA-4588 marlin "root" package.json depdencies too old to
 build on x86_64 Reviewed by: Josh Wilsdon <josh@wilsdon.ca> Approved by: Josh
 Wilsdon <josh@wilsdon.ca>

---
 package.json | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/package.json b/package.json
index 94345f5..8fc41a3 100644
--- a/package.json
+++ b/package.json
@@ -16,13 +16,13 @@
 	},
 	"dependencies": {
 		"assert-plus": "0.1.2",
-		"bunyan": "0.20.0",
-		"dtrace-provider": "0.2.8",
+		"bunyan": "1.8.12",
+		"dtrace-provider": "0.8.8",
 		"extsprintf": "1.0.1",
 		"jsprim": "0.5.0",
 		"lstream": "0.0.3",
 		"mahi": "git+https://github.com/joyent/node-mahi.git#master",
-		"moray": "3.0.0",
+		"moray": "3.7.0",
 		"node-uuid": "1.4.0",
 		"posix-getopt": "1.0.0",
 		"retry": "0.6.0",
-- 
2.21.0

